docs/source/install/monitoring_stack.rst

```diff
@@ -10,12 +10,12 @@ For evaluation, you can run Scylla Monitoring Stack on any server (or laptop) th
 
 .. include:: min-prod-hw.rst
 
-.. _`Scylla Prometheus API`: monitoring_apis#prometheus
+.. _`Scylla Prometheus API`: /reference/monitoring_apis#prometheus
```

Comment:
Relative URLs should not start with / (slash).

**Possible solution:** Delete this line, and in line 5, use a cross reference link instead:

```
* This is an example of an :ref:`Internal Link <link-format-examples>`. This is an internal cross reference. It requires a bookmark.
```

For more information, see [Links](https://sphinx-theme.scylladb.com/stable/examples/links.html).